sample_apps/rso_sample_apps/rust/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "riot_rso_sample_app"
```

Comment:
Add a rust-toolchain.toml to make sure the user who has rustup uses the correct toolchain version: https://gh.riotgames.com/riotgamesapi/gitchonk/blob/main/rust-toolchain.toml